### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by martinlatrille**
#### Fixs:
- fix drop-shadow of messenger icon
- add validation for capture email input
- font-size of capture email input is now 15px
- fix size of first message of appmaker when name was too long
- fix z-index to always be above everything
- edit mediaquery breakpoints to always display complete chat widget

### CODE REVIEW

File: src/js/components/email-capture.jsx [hunk 1]
```diff
@@ -74,8 +74,9 @@ class EmailCaptureComponent extends Component {
                     <input
                         placeholder={placeholder}
                         onChange={(e) => this.setState({email: e.target.value})}
+                        pattern='^[\w\.\-\+]+@[\w\.\-]+\.\w+$'
```
Comment (LouisBarranqueiro) [inline src/js/components/email-capture.jsx]: `type="email"` would be enough for desktop and better for mobile

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
         window.SmoochOptions = {
             serviceUrl: 'ROOT_URL',
-            appToken: '85qe4lk1c78cc22rzzyogjf55',
+            appToken: '6iv99qftiz6tut11b75zply45',
```
Comment (LouisBarranqueiro) [inline src/index.html]: reminder?

Comment (martinlatrille) [reply src/index.html]: Nah I change it everytime, the old one is one of an app which is deleted (local Smooch app)

File: src/js/components/email-capture.jsx (hunk 1 above)
Comment (martinlatrille) [reply src/js/components/email-capture.jsx]: What do you mean? Bc right now it's exactly our regex for the backend so it's better no? How is it worse for mobile?

File: src/stylesheets/main.less [hunk 3]
```diff
@@ -97,7 +97,7 @@
         margin-bottom: @widget-vertical-spacing;
         box-shadow: 0 36px 64px rgba(0,0,0,0.09);
         //border: 1px solid rgba(0,0,0,0.15);
-        z-index: 9998; // 1 less than Stripe Checkout
+        z-index: 999999999 !important;
```
[the remark this answers is not in the document]
Comment (martinlatrille) [reply src/stylesheets/main.less]: 👋

File: src/stylesheets/messenger-button.less [hunk 4]
```diff
@@ -17,7 +17,7 @@
     border-radius: 50%;
 
     box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26);
-    z-index: 9998; // 1 less than Stripe Checkout
+    z-index: 999999998 !important;  // one less than the global container, so that it's below it in mobile mode
```
Comment (LouisBarranqueiro) [inline src/stylesheets/messenger-button.less]: use variables for all theses z-indexes. easier to manipulate across the project.

File: src/js/components/email-capture.jsx (hunk 1 above)
Comment (LouisBarranqueiro) [reply src/js/components/email-capture.jsx]: `type="email"` integrate a regex and also on mobile  `@`, `.com` characters are enable on the first keyboard.

File: src/stylesheets/messenger-button.less (hunk 4 above)
Comment (martinlatrille) [reply src/stylesheets/messenger-button.less]: Done.

File: src/js/components/email-capture.jsx (hunk 1 above)
Comment (martinlatrille) [reply src/js/components/email-capture.jsx]: Done having both.

File: src/stylesheets/variables.less [hunk 5]
```diff
@@ -93,3 +93,5 @@
 
 // Gorgias variables
 @gorgias-active-bluegreen: #06D79C;
+@chat-z-index: 999999999;
+@lower-z-index: 999999998;
```
Comment (LouisBarranqueiro) [inline src/stylesheets/variables.less]: what "lower" means? "chat-wrapper" or something like this no?

**Pull request merged**